### ISSUE TITLE
Convert to JavaScript and host on github.io

### ISSUE THREAD

**Issue by dlh3** [dlh3/query-strava#6]
Maybe I can talk @elcurto99 into helping out.  ;)

**Comment by elcurto99** [comment]
Oh yeah?

**Comment by dlh3** [comment]
Yes, I thought I might be able to entice him with elephant avatars and post-run beers, with a side of opportunity.  Would you capture it?  Or just let it slip?

**Comment by dlh3** [comment]
Yes, I thought I might be able to entice him with elephant avatars and
post-run beers, with a side of opportunity.  Would you capture it?  Or just
let it slip?
